packages/documentation/copy/en/reference/Utility Types.md

```diff
@@ -320,6 +320,8 @@ Released:
 
 Constructs a tuple type from the types used in the parameters of a function type `Type`.
 
+For overloaded functions, this will be the parameters of the _last_ signature; see [Inferring Within Conditional Types](docs/handbook/2/conditional-types.html#inferring-within-conditional-types).
```

Comment:
```suggestion
For overloaded functions, this will be the parameters of the _last_ signature; see [Inferring Within Conditional Types](/docs/handbook/2/conditional-types.html#inferring-within-conditional-types).
```